### PR TITLE
Update net40 and net45 to net452

### LINKED ISSUE #1835: Remove target frameworks v4 and v4.5
### Feature description
Remove target frameworks v4 and v4.5 so that the project can be build in Visual Studio 2022. From https://docs.microsoft.com/en-gb/dotnet/framework/install/guide-for-developers:
> Starting with Visual Studio 2022, Visual Studio no longer includes .NET Framework components for .NET Framework 4.0 - 4.5.1 because these versions are no longer supported. Visual Studio 2022 and later versions can't build apps that target .NET Framework 4.0 through .NET Framework 4.5.1. To continue building these apps, you can use Visual Studio 2019 or an earlier version.

### PR DESCRIPTION
**Pull request by BobLd**
Fixes  #1835 (previous PR was #1836)

### Checklist

- [X] I have included examples or tests - no need
- [X] I have updated the change log
- [X] I am listed in the CONTRIBUTORS file
- [X] I have cleaned up the commit history (use rebase and squash)

### Changes proposed in this pull request:

- Update `net40`/`net45` to `net452`
As an aside from https://docs.microsoft.com/en-GB/lifecycle/faq/dotnet-framework
> Support for .NET Framework 4, 4.5, and 4.5.1 ended on January 12, 2016.
> Support for .NET Framework 4.5.2, 4.6, and 4.6.1 will end on April 26, 2022. Customers and developers must have completed the in-place update to .NET Framework 4.6.2 or later by April 26, 2022 to continue receiving technical support and security updates.

@oxyplot/admins, @VisualMelon

### CODE REVIEW

File: Source/OxyPlot.Wpf/XpsExporter.cs [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 namespace OxyPlot.Wpf
 {
-#if !NET40
```
Comment (BobLd) [inline Source/OxyPlot.Wpf/XpsExporter.cs:12]: To review - I'm not sure replacing `#if !NET40` by `#if !NETFRAMEWORK` is correct

Comment (VisualMelon) [reply Source/OxyPlot.Wpf/XpsExporter.cs:12]: Am I wrong to think we can just remove this check altogether? (not checked, in the middle of other things)

Comment (BobLd) [reply Source/OxyPlot.Wpf/XpsExporter.cs:12]: fair point, checking that

Comment (BobLd) [reply Source/OxyPlot.Wpf/XpsExporter.cs:12]: I think it's better to leave it like that, `System.Windows.Xps` is not available in `net452` but is in `netcoreapp3.1`. So it seems checking for `#if !NETFRAMEWORK` is correct

**Comment by BobLd** [comment]
I just realised I removed too much of the compatibility extensions, strangely VS didn't say anything. GOing to fix that now

**Comment by BobLd** [comment]
Should be good to go

**Comment by Jonarw** [comment]
Thanks for the PR and sorry for taking so long... I don't find a lot of time atm to look at this repo.

There are two more projects (`Examples>WPF>SimpleDemo` and `Examples>WindowsForms>WindowsFormsDemo`) which still have `net40` and `net45` targets. I think we could drop them altogether and only leave the netcore3.1 target.

Other than that looks good to me :)

**Comment by BobLd** [comment]
@Jonarw, thanks for pointing that out, it's now fixed! I've just replaced `net40/45` by `net452`

**Review by Jonarw — APPROVED** [review]

**Comment by VisualMelon** [comment]
@Jonarw thanks for picking this up!

**Pull request merged**